### PR TITLE
Bump up ks-devops charts for next release

### PR DESCRIPTION
**Pull request by JohnNiang**
Signed-off-by: John Niang <johnniang@fastmail.com>

### CODE REVIEW

File: charts/ks-devops/Chart.yaml [hunk 1]
```diff
@@ -15,12 +15,12 @@ type: application
...
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.8
+version: 0.1.9
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: v3.2.0
+appVersion: v3.2.1
```
Comment (LinuxSuRen) [inline charts/ks-devops/Chart.yaml:23]: Actually, we still don't have v3.2.1 of ks-devops (apiserver and controller). How about releasing a new version v3.2.1-rc2

Comment (JohnNiang) [reply charts/ks-devops/Chart.yaml:23]: Got it.

**Review by LinuxSuRen — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**